src/test/java/org/broadinstitute/consent/http/ConsentAssociationTest.java

```diff
@@ -1,20 +1,19 @@
 package org.broadinstitute.consent.http;
 
+import static io.dropwizard.testing.FixtureHelpers.fixture;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
What approach are you using for formatting code? I used the default Reformat Code with the Optimize Imports option checked in IntelliJ, but a lot of this code seems to be formatted differently.

Reply:
Intellij did all of the formatting, I didn't do it at all.

Reply:
Ok, I don't want to block merging this PR. I was curious.